hw/top_earlgrey/rdc/rdc_waivers.tcl

```diff
@@ -130,3 +130,42 @@ set_rule_status -rule E_RST_METASTABILITY -status Waived \
   -expression {(SourceReset=="POR_N") && \
     (ResetFlop=~"*.u_clkmgr_aon.*usb_peri_sw_en_sync*") && \
     (MetaStableFlop=~"*.u_clk_usb_peri_cg.*")}
+
+# EN sync to EN Latch in CG
+# (io_div4|io_div2|io|usb)_root_ctrl
+create_view_criteria -name PorNRootCtrlCg -rule E_RST_METASTABILITY \
```

Comment:
just to confirm, this first category is basically us ignoring POR induced metability right? since everything is reset? let me know if i misunderstood.

Reply:
Yes. The waivers below is to ignore metastability when PorN is asserted.

This specific line is to create a view group to show what paths were waived for analysis later.